### PR TITLE
Submit improvements

### PR DESCRIPTION
**Pull request by LouiseBonnard**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,78 +1,76 @@
...
-5. Change the files in your branch. Be sure to follow the coding standards.
-6. Push your changed branch to your fork in your GitHub account.
-7. Create a pull request for your changes on the PrestaShop project. If you need help to make a pull request, read the Github help page about creating pull requests.
-8. Wait for one of the product manager either to include your change in the codebase, or to comment on possible improvements you should make to your content.
+1. Fork the PrestaShop project to your Github account.
+2. Clone your fork to your local machine.
+3. Create a branch in your local clone for your changes.
+4. Change the files in your branch. Make sure they meet our [coding standards](https://devdocs.prestashop.com/1.7/development/coding-standards).
+5. Push the changed branch to your fork in your GitHub account.
+6. Create a pull request for your changes on the PrestaShop project. If you need help to make a pull request, read the GitHub help page about creating pull requests.
```
Comment (LouiseBonnard) [inline CONTRIBUTING.md]: Link missing.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,78 +1,76 @@
...
 
 
-### Adding missing specs
+### Add missing specifications
 
-We add and complete the specifications progressively, thus some features specifications are missing for the moment.
-You can add the specifications to these existing features.
-These specs should define the current behaviors in the latest Stable version.
-To do so you can follow the structure used on the template at the end of the end of this page, and see some examples in the repository or in the [templates folder](/spec-templates).
+We add and complete the specifications progressively, although a lot is missing for the moment. You can trust us, we know.
```
Comment (sam-pires) [inline CONTRIBUTING.md]:
```suggestion
We add and complete the specifications progressively, although a lot is missing for the moment. You can trust us, we are aware of it.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,78 +1,76 @@
...
 ### If you feel comfortable working with Git and forking the project
 
-2. Fork the PrestaShop project to your Github account.
-3. Clone your fork to your local machine.
-4. Create a branch in your local clone for your changes.
-5. Change the files in your branch. Be sure to follow the coding standards.
-6. Push your changed branch to your fork in your GitHub account.
-7. Create a pull request for your changes on the PrestaShop project. If you need help to make a pull request, read the Github help page about creating pull requests.
-8. Wait for one of the product manager either to include your change in the codebase, or to comment on possible improvements you should make to your content.
+1. Fork the PrestaShop project to your Github account.
```
Comment (sam-pires) [inline CONTRIBUTING.md]:
```suggestion
1. Create your GitHub account, if you do not have one already.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,78 +1,76 @@
...
+2. Clone your fork to your local machine.
```
Comment (sam-pires) [inline CONTRIBUTING.md]:
```suggestion
2. Fork the PrestaShop project to your Github account.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,78 +1,76 @@
...
+3. Create a branch in your local clone for your changes.
```
Comment (sam-pires) [inline CONTRIBUTING.md]:
```suggestion
3. Clone your fork to your local machine.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,78 +1,76 @@
...
+4. Change the files in your branch. Make sure they meet our [coding standards](https://devdocs.prestashop.com/1.7/development/coding-standards).
```
Comment (sam-pires) [inline CONTRIBUTING.md]:
```suggestion
4. Create a branch in your local clone for your changes.
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,78 +1,76 @@
...
+5. Push the changed branch to your fork in your GitHub account.
```
Comment (sam-pires) [inline CONTRIBUTING.md]:
```suggestion
5. Change the files in your branch. Make sure they meet our [coding standards](https://devdocs.prestashop.com/1.7/development/coding-standards).
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (sam-pires) [inline CONTRIBUTING.md]:
```suggestion
6. Push the changed branch to your fork in your GitHub account.
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,78 +1,76 @@
...
+6. Create a pull request for your changes on the PrestaShop project. If you need help to make a pull request, read the GitHub help page about creating pull requests.
+7. Wait for one of the product manager to either include your change in the codebase or comment on possible adjustements to its content.
```
Comment (sam-pires) [inline CONTRIBUTING.md]:
```suggestion
7. Create a pull request for your changes on the PrestaShop project. If you need help to make a pull request, read the GitHub help page about [creating pull requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,78 +1,76 @@
...
```
Comment (sam-pires) [inline CONTRIBUTING.md]:
```suggestion
8. Wait for one of the product managers to either include your change in the codebase or comment on possible adjustments to its content.

```

File: CONTRIBUTING.md (hunk 1 above)
Comment (sam-pires) [reply CONTRIBUTING.md]: solved in the sugestion

**Comment by colinegin** [comment]
Hello @LouiseBonnard ,

What's the status of this PR ?

**Comment by LouiseBonnard** [comment]
@sam-pires, you can merge now. ;-)

**Comment by matthieu-rolland** [comment]
There's just a small conflict to solve before merging. 👍

**Pull request merged**